test/test_action_card.py

```diff
@@ -0,0 +1,69 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
+from .test_helpers import construct_mock_context
+
+from pyndemic.action_card import (GovernmentGrantActionCard,
+                                  QuietNightActionCard)
+
+
+class GovernmentGrantActionCardCase(TestCase):
+    def setUp(self):
+        self._ctx = construct_mock_context()
+        self.mock_game = self._ctx['controller']().game
+
+    def test_init(self):
+        card = GovernmentGrantActionCard()
+        self.assertEqual("Government Grant", card.name)
+        self.assertIsNone(card.colour)
+
+    def test_check_playable(self):
+        card = GovernmentGrantActionCard()
+
+        self.mock_game.city_map = {}
+        playable = card.check_playable("London")
+        self.assertFalse(playable)
+
+        self.mock_game.city_map["London"] = MagicMock()
+        self.mock_game.city_map["London"].has_lab = True
+        playable = card.check_playable("London")
+        self.assertFalse(playable)
+
+        self.mock_game.city_map["London"] = MagicMock()
+        self.mock_game.city_map["London"].has_lab = False
+        playable = card.check_playable("London")
+        self.assertTrue(playable)
+
+    def test_on_play(self):
+        card = GovernmentGrantActionCard()
+        self.mock_game.city_map = {"London": MagicMock()}
+        self.mock_game.city_map["London"].has_lab = False
+        result = card.on_play("London")
+        self.assertTrue(result)
```

Comment:
I think we also need to ensure here that the lab was built in London.
```suggestion
        self.assertTrue(result)
        self.assertTrue(self.mock_game.city_map["London"].has_lab)
```